src/components/Input/TextArea/TextArea.test.tsx

```diff
@@ -0,0 +1,37 @@
+/* External dependencies */
+import React from 'react'
+
+/* Internal dependencies */
+import { render } from '../../../utils/testUtils'
+import TextArea, { TEXT_AREA_TEST_ID } from './TextArea'
+import { TextAreaProps } from './TextArea.types'
+
+// TEST CODE 강화 필요
```

Comment:
```suggestion
// TODO: TEST CODE 강화 필요
```
혹은 이 PR 에서 웬만하면 TestCode 보강을 부탁드리겠습니다.

Reply:
여기서 처리하겠습니다